geopandas/tools/tests/test_sjoin.py

```diff
@@ -117,6 +117,23 @@ def test_left(self, op, dfs):
 
         assert_frame_equal(res, exp)
 
+    def test_empty_join(self):
+            # Check empty joins on left
```

Comment:
minor nit, but the indentation seems a bit too much here?

Reply:
yep, no, was not intended. thanks for catching this.